### PR TITLE
Add scripts to verify anchors

### PR DESCRIPTION
**Pull request by yikeke**
### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/docs-tidb-operator/pull/24/, https://github.com/pingcap/docs-tidb-operator/pull/210/

### CODE REVIEW

**Comment by yikeke** [comment]
Could you help fix the dead link anchors reported in [here](https://github.com/pingcap/docs-dm/pull/146/checks?check_run_id=834042418)? I'm not sure how to fix them. @WangXiangUSTC

File: en/relay-log.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ An example of the directory structure of the local storage for a relay log:
 
     - `subdir` is named `<Upstream database UUID>.<Local subdir serial number>`.
 
-    - After [a switch between primary and secondary instances](cluster-operations.md#switch-between-primary-and-secondary-instances) in the upstream, DM-worker generates a new `subdir` directory with an incremental serial number.
+    - After a switch between primary and secondary instances in the upstream, DM-worker generates a new `subdir` directory with an incremental serial number.
```
Comment (WangXiangUSTC) [inline en/relay-log.md:40]: 中文文档也没有链接，因此这里删掉了

Comment (yikeke) [reply en/relay-log.md:40]: 好

Comment (yikeke) [reply en/relay-log.md:40]: review 后如果没问题，相总帮忙点个赞吧

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -9,10 +9,15 @@ jobs:
...
+        node-version: '12'
     - name: Markdown lint
       uses: avto-dev/markdown-lint@v1
       with:
         config: './.markdownlint.yaml'
         args: '.'
+    - name: Verify links
+      run: ./hack/verify-links.sh
+    - name: Verify link anchors
+      run: ./hack/verify-link-anchors.sh
```
Comment (yikeke) [inline .github/workflows/ci.yaml:23]:
```suggestion
      run: ./hack/verify-link-anchors.sh 
```

File: en/relay-log.md (hunk 1 above)
Comment (toutdesuite) [reply en/relay-log.md:40]: 英文相关修改历史https://github.com/pingcap/docs-dm/commit/c846c9bcc831eb6051c7a3ce3a927488b1368c0c

**Pull request merged**